recipes/libdispatch/bld.bat

```diff
@@ -0,0 +1,22 @@
+setlocal EnableDelayedExpansion
+
+cmake %CMAKE_ARGS% ^
+    -G Ninja ^
+    -B build ^
+    -D BUILD_SHARED_LIBS=YES ^
+    -D CMAKE_BUILD_TYPE=Release ^
+    -D CMAKE_C_COMPILER=clang-cl ^
+    -D CMAKE_CXX_COMPILER=clang-cl ^
+    -D CMAKE_MT=mt ^
+    -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+    -D CMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+    -D CMAKE_SYSTEM_NAME=Windows ^
+    -D CMAKE_SYSTEM_PROCESSOR=x86_64 ^
```

Comment:
This should be set via environment variables already and should be omitted in preparation for win-arm64
```suggestion
```